doc/faq.rst

```diff
@@ -128,6 +128,10 @@ Mesh-Related Tasks
    ``<dimension>`` is the topological dimension of cells that should be in the
    mesh. For example, ``-d 2`` stores only the 2D cells.
 
+#. Use ``msh22`` format instead of the default ``msh4`` when generating a mesh 
+   with ``gmsh``::
```

Comment:
(the ``msh4`` format does not work well with the -2 option)??

Reply:
OK, I would write "It is suggested to use ..., msh22 is more reliable and foolproof."

Reply:
OK!

Reply:
Updated.

Reply:
LGTM, just fix the line length, please :)

Reply:
Fixed :)